lib/matplotlib/animation.py

```diff
@@ -907,6 +908,8 @@ class Animation:
     """
 
     def __init__(self, fig, event_source=None, blit=False):
+        self._anim_was_inited = False
```

Comment:
```suggestion
        self._draw_was_inited = False
```
(or maybe `_draw_was_started`, which is a bit less technical, but IMHO captures better what we want to express).